core/typeSugar.ml

```diff
@@ -3603,35 +3644,36 @@ let rec type_check : context -> phrase -> phrase * Types.datatype * Usage.t =
                           let fieldtype = Types.for_all (pqs, t) in
 
                           (* really we just need to unify the presence
-                             variable with `Presence, but our griper
+                             variable with Presence, but our griper
                              interface doesn't currently support
                              that *)
```

Comment:
Hmmm... looking more closely at this unification, it seems completely unnecessary. It just unifies a data type with an unconstrained type variable that is immediately discarded. It will always succeed and serves no purpose. I'll remove this code.